modules/rosa-sts-account-wide-roles-and-policies.adoc

```diff
@@ -730,6 +730,44 @@ The account number present in the `sts_installer_trust_policy.json` and `sts_sup
 ----
 ====
 
+[discrete]
+[id="rosa-sts-account-wide-roles-and-policies-example-cli-output-for-policies-attached-to-a-role_{context}"]
+==== Example CLI output for policies attached to a role 
+
+When a policy is attached to a role, the ROSA CLI displays confirmation output.
+
+For the target role with policy attached, ROSA CLI outputs the role name and the console URL of it. 
+
+.Target role with policy attached example output
+[source,terminal]
+----
+I: Attached trust policy to role 'testrole-Worker-Role(https://console.aws.amazon.com/iam/home?#/roles/testrole-Worker-Role)': ******************
+----
+
+If the attached policy is a trust policy, ROSA CLI outputs the content of this policy.
```

Comment:
it's better to mention the policy type explicitly like below:

if a policy is a **trust policy**, then: output content
if a policy is a **permission policy**, then:
     if this policy is aws managed policy, then: output name+public link
     if this policy is not a managed policy, then: output arn